src/tasks/index.js

```diff
@@ -81,6 +81,21 @@ class TaskProvider {
    * @returns { AsyncGenerator<T> | Generator<T> | AsyncIterable<T> | Iterable<T> }
    */
   getTasks () { return [] }
+
+  /**
+   * Retrieve a listing of tasks from the service.
+   *
+   * This function does not accept any parameters.
+   * If you want to configure its behaviour, you should configure the object that contains it instead.
+   *
+   * Note that this function is an async generator. This means that you may use asynchronous calls inside it.
+   * Additionally, because it is a generator function, you use a yield statement instead of a return statement.
+   * this is meant to accommodate APIs that paginate their results.
+   *
+   * TODO: do we want to allow passing queries, such as "tasks due within X days" here?
+   * @returns { AsyncGenerator<T> | Generator<T> | AsyncIterable<T> | Iterable<T> }
+   */
+  getTask () { return [] }
```

Comment:
Is this copy-pasta? I don't see it used anywhere and the comment seems to be a duplicate of `getTasks()`.

Reply:
Yep! Looks it to me - probably a dead end from something I was working on at some point